RELEASING.adoc

```diff
@@ -1,80 +1,210 @@
 = Releasing Kiali
 
-This document gives the steps required to perform a release of Kiali. This document is only useful to Kiali release managers.
+This document covers the specifics of releasing Kiali, and is meant as a guide
+for Kiali release managers.
+
+The summary of the process is the following:
+
+* Create a new JIRA issue for the release
+* Make sure Getting Started docs are complete
+* Update Kiali UI version
+* Publish the API Docs
+* Update Makefile
+* Create the Release PR
+* Tag Git Repo
+* Notify Istio and Openshift Istio Teams for changes:
```

Comment:
I think it would be good to add links where/how to notify

Reply:
Same as the last comment, this is just a summary on what to do. The steps themselves are on the relevant section.